src/diamonds/nayms/AppStorage.sol

```diff
@@ -17,6 +17,8 @@ struct AppStorage {
     mapping(bytes32 => bytes32) objectParent; // objectId => parentId
     mapping(bytes32 => bytes32) objectDataHashes;
     mapping(bytes32 => bytes32) objectTokenSymbol;
+    mapping(bytes32 => bytes32) objectTokenName;
```

Comment:
name and symbol are typically stored as string type, can we confirm that we want to store them as bytes32?

Reply:
Symbol was already a bytes32, so I just used the same type for the name, but I agree it feels more appropriate to store a string.